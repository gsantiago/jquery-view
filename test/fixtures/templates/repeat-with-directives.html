<ul>
  <li :repeat="user in users">
    <span>NÂº: {{$index + 1}}</span>
    <strong :bind="user.name"></strong>
    <a :href="'mailto:' + user.email" :show="user.email">Send email</a>
  </li>
</ul>