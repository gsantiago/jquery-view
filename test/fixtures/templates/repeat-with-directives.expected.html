<ul>
  <li>
    <span>Nº: 1</span>
    <strong>Guilherme</strong>
    <a href="mailto:guilherme@email.com">Send email</a>
  </li><li>
    <span>Nº: 2</span>
    <strong>Zilla</strong>
  </li><li>
    <span>Nº: 3</span>
    <strong>Junin</strong>
    <a href="mailto:junin@email.com">Send email</a>
  </li>
</ul>